src/components/Profile/components/VictoryPromise/components/VictoryPromiseIcons.tsx

```diff
@@ -23,8 +22,8 @@ const VictoryPromiseIcon = ({
   category: VictoryPromiseCategory;
   active: boolean;
 }) => {
-  const primary = active ? Colors[category] : gordonColors.primary.blue;
-  const secondary = active ? gordonColors.neutral.grayShades[50] : gordonColors.neutral.gray;
+  const primary = active ? Colors[category] : '#014983';
+  const secondary = active ? '#ffffff' : '#CCCCCB';
```

Comment:
Same thing here with the reused colors, if might be better to save the reused ones to a variable.

Also, just to double check, 'var(--mui-...)' inside quotes didn't work here, right?